repeat/p5js/hidden-circles/index.js

```diff
@@ -0,0 +1,46 @@
+var dis = 20;
+
+function setup() {
+  createCanvas(500, 500);
+  background(255);
+}
+
+function drawLines(count, distance) {
+  distance = distance || 20;
+
+  for (var i = 0; i < count; i++) {
+     var x = i * distance;
+
+    line(x, 0, x, 500);
+  }
+}
+
+function drawEllipsis(height) {
+  strokeWeight(10);
+  stroke(255, 255, 255);
+  noFill();
+  ellipse(250, 250, height);
+}
+
+function draw() {
+  stroke(0);
+  strokeWeight(2);
+  drawLines(100, dis);
+
+  drawEllipsis(100)
+
+  fill(255);
+  ellipse(250, 250, 35);
+
+  drawEllipsis(150);
```

Comment:
DRY (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) does not necessarily mean don't write the same thing a couple of times. Here for instance the `drawEllipsis` function can be seen as a sub-component which is used a couple of times. So the code is actually dry. Everything that is needed to draw an ellipsis is encapsulated in this method. Using it a couple of times does not violate that rule. 

Also I doubt that a loop like

```jsx
[150, 240, 300, 340, 400, 440, 500].forEach((height) => drawEllipsis(height))
``` 

Would make this code easier to understand / better maintainable.